elements/pl-diagram/diagram-editor.js

```diff
@@ -0,0 +1,422 @@
+/**
+ * Copyright (c) 2006-2020, JGraph Ltd
+ * Copyright (c) 2006-2020, Gaudenz Alder
+ *
+ * Usage: DiagramEditor.editElement(elt) where elt is an img or object with
+ * a data URI src or data attribute or an svg node with a content attribute.
+ *
+ * See https://jgraph.github.io/drawio-integration/javascript.html
+ */
+function DiagramEditor(config, ui, done, initialized)
```

Comment:
I had to heavily modify this file to integrate it, since it is mostly a wrapper for accessing the draw.io API, so I don't think I can install this through `npm`.